spec/requests/api/v1/admin/admin_index_spec.rb

```diff
@@ -0,0 +1,93 @@
+RSpec.describe "GET '/api/v1/admin/articles" do
+  let!(:editor) { create(:user, role: "editor")}
```

Comment:
have these users and articles be created in their respective describe blocks

Reply:
where they are used

Reply:
Is it not more DRY to create them for the whole test? Since we are using the same articles for several describe blocks?